client/src/Components/Pages/Dashboard/ActiveJobs/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Input from '../../../CommonComponents/Inputs';
+import Button from '../../../CommonComponents/Button';
+
+import "./style.css";
+
+export default function index(props){
```

Comment:
I _think_ you don't need `index` here and you could just write `export default ( props ) => {`, if you want.

You can destructure `action` right inside the parenthesis, too, if you really want to shorten things:
`export default ({ action }) => {`

Reply:
Great, I will modify all the components in this way and make new PR for the updates.